function-maven-plugin/pom.xml

```diff
@@ -44,7 +45,7 @@
     <dependency>
       <groupId>com.google.cloud.functions.invoker</groupId>
       <artifactId>java-function-invoker</artifactId>
-      <version>1.1.2-SNAPSHOT</version>
+      <version>1.1.1</version>
```

Comment:
I guess this version bump is done by release-prepare while testing? We should probably revert back

Reply:
This is not done by release-prepare. I added this to remove the interdependency between function-maven-plugin and invoker.

Reply:
Got it. Thanks for clarification.